global.json

```diff
@@ -1,5 +1,6 @@
 {
     "projects": [
-        "src"
+        "src",
+        "tools"
```

Comment:
Will this cause NuGet packages to get created for the tools projects? The reason I created the tools directory in the first place was to avoid that.